spec/requests/entities_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'Entities', type: :request do
+  include Devise::Test::IntegrationHelpers
+  before do
+    @user = User.create(name: 'Hanningtone', password: 'password', email: 'han@gmail.com')
+    @group = @user.groups.create(name: 'Birthday', icon: 'https://google.com')
+    sign_in @user
+  end
+
+  describe 'GET /index' do
+    it 'returns http success' do
+      get group_entities_path(@group)
+      expect(response).to have_http_status(:success)
+    end
+  end
+end
```

Comment:
- [ ] Nice adding units tests but you should also add integration tests for most components using capybara to make sure everything works fine so I encourage you to do that 
![Screen Shot 2022-10-11 at 17 07 20](https://user-images.githubusercontent.com/61790587/195143532-917fd136-3720-4bc1-80b6-51b6981fcea7.png)